tests/bfd/test_bfd.py

```diff
@@ -220,16 +226,19 @@ def check_ptf_bfd_status(ptfhost, neighbor_addr, local_addr, expected_state):
     for line in bfd_state:
```

Comment:
>for [](http://example.com/codeflow?start=4&length=3)

Will you return something if the for loop not return in the middle?

Reply:
Thanks. I have updated it to return empty string to represent no value found and test would fail.